sfyra/cmd/sfyra/cmd/options.go

```diff
@@ -67,7 +67,7 @@ func DefaultOptions() Options {
 		TalosKernelURL: fmt.Sprintf("https://github.com/talos-systems/talos/releases/download/%s/vmlinuz-amd64", TalosRelease),
 		TalosInitrdURL: fmt.Sprintf("https://github.com/talos-systems/talos/releases/download/%s/initramfs-amd64.xz", TalosRelease),
 
-		CoreProvider:            "cluster-api:v0.4.3",
+		CoreProvider:            "cluster-api:v1.0.0",
```

Comment:
should we skip locking the version of CAPI here? there's no bug now in CAPI which installed wrong version not matching the contract

Reply:
If all 1.x.x will be backward compatible then yeah.